### ISSUE TITLE
Does FieldResourceRepository removed in 2.4.0? 

### ISSUE THREAD

**Issue by kaleeswaran393** [katharsis-project/katharsis-core#333]
Hi Team,

    I want to insert  Parent and Child Object in single request and I should able to bind task and project json data  using parent repository Save method.is it possible to create or update child object attribute which are included in the included tag .

   {
  "data": {
    "type": "task",
    "id": "1",
    "attributes": {
      "task-name": "Task Name"
    },
    "relationships": {
      "project": {
        "data": [
          {
            "type": "project",
            "id": "1"
          }
        ]
      }
    }
  },
  "included": [
    {
      "type": "project",
      "id": "1",
      "attributes": {
        "project-name": "Analysis"
      }
    }
  ]
}
  Basically I should able the bind the above json data into task object using Task Repository. I don't have Project repository.

Thanks,
Kaleeswaran K
